onnx_xla/operator_registry.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include "onnx/common/ir.h"
+#include "onnx/common/ir_pb_converter.h"
+#include "onnx/onnxifi.h"
+#include "tensorflow/compiler/xla/client/xla_client/xla_builder.h"
+
+#include <functional>
+
+namespace onnx_xla  {
+
+  using xla::Literal;
+  using xla::ShapeUtil;
+  using xla::Shape;
+  using xla::primitive_util::NativeToPrimitiveType;
+  using xla::XlaOp;
+  using xla::XlaBuilder;
+  using xla::LiteralBase;
+  using xla::StatusOr;
+
+  using ONNX_NAMESPACE::Tensor;
+  using ONNX_NAMESPACE::Value;
+  using ONNX_NAMESPACE::Dimension;
+  using ONNX_NAMESPACE::Graph;
```

Comment:
this doesn't seem to be used in the registry?

Reply:
Btw, it's better to use `::ONNX_NAMESPACE` to avoid namespace collision.

Reply:
Though Graph is not used now, I kept it here because an operator may have type subgraph or we could potentially be interested in the owning graph. Is this reasonable or should I remove?

Reply:
@varunjain99 I'd prefer to remove them for now and add them when they are used (to make the current code and the future diff that starts using additional things clearer).